containers/docker-packager-base/Dockerfile

```diff
@@ -0,0 +1,43 @@
+#
+#    Copyright 2020 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+FROM python:3.6
+
+ARG MINICONDA_URL=https://repo.anaconda.com/miniconda/Miniconda3-4.7.12-Linux-x86_64.sh
```

Comment:
You're using version 4.7.12 here, but 4.5.11 in `odahu-flow` robot-test, for example. Is it ok for us?

Reply:
Thanks, fixed in odahu-robot-tests